src/lib/sensor.js

```diff
@@ -0,0 +1,38 @@
+const Subdevice = require('./subdevice')
+
+class Sensor extends Subdevice {
+  constructor (opts) {
+    super({ sid: opts.sid, type: 'sensor' })
+  }
+
+  _handleState (state) {
+    super._handleState(state)
+
+    // all fields come at once at first but one-by-one later
+    if (state.temperature) {
+      this._temperature = state.temperature / 100
+    }
+    if (state.humidity) {
+      this._humidity = state.humidity / 100
+    }
+
+    if (this._timeout) {
+      clearTimeout(this._timeout)
+    }
+
+    this._timeout = setTimeout(() => {
```

Comment:
Why? I think it's not an issue if two updates are triggered within a very short interval. Don't you agree?